projects/projen.io/js/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://github.com/projen/projen/archive/refs/tags/v{{version.raw}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: projen/projen
+  strip: /^v/
+
+dependencies:
+  nodejs.org: ^22
+  classic.yarnpkg.com: ^1
+  git-scm.org: "*"
```

Comment:
is this required to run? or just to test? or is it a companion?

Reply:
Same as with yarn, projen executes git during runtime when creating or mainting projects. Can you clarify what's the difference between companion and dependency?

Reply:
Generally, a program requires a dependency to execute, most often libraries. A companion is usually another program, which may be optional, that provides functionality.

Companion functionality is newer than dependency functionality, and there are, guaranteed, many violations of this split in the repo.